src/themes/README.md

```diff
@@ -0,0 +1,36 @@
+# Creating custom themes
+
+Base UI ships by default with the `Light` theme. We understand that in some cases
+you want to change the look and feel of the components. To do so, follow the steps
+outlined here
+
+## Define your primitives
+
+Primitives are used to define the colors and fonts for the components. An example for
+primitives can be found in the [light theme](https://github.com/uber-web/baseui/blob/master/src/themes/light-theme.js).
+
+## Define any additional theme overrides
+
+Theme overrides are used to change values in the theme like paddings or borders.
+To learn more about what you can override, check out the [creator](https://github.com/uber-web/baseui/blob/master/src/themes/creator.js).
+
+## Putting it all together
+
+In practice, you can define your custom theme using this approach:
+
+```js
+import createTheme from 'baseui/themes/creator'
+
+import {primitives} from 'baseui/themes/light-theme';
+
+const yourTheme = createTheme({
+  ...primitives,
```

Comment:
Currently we have only primitives, but if we may have other elements, they might not be covered with this example.
And seems like it should be
```js
const yourTheme = createTheme({
  ...{ 
         ...primitives,
         primaryFontFamily: '"Comic Sans MS", cursive, sans-serif'
      }
})
```

Reply:
What do you mean by other elements? I guess if we break this API, we'll have to update the readme anyways - what am I missing?

Reply:
I thought the whole theme object is
```js
{
   primitives,
   // other properties
}
```
according to this file src/themes/light-theme.js
If so, we override here only primitives features

Reply:
that's not the case - checkout out the theme object too, that's the second param of the function

Reply:
ok